backend-tests/run

```diff
@@ -1,28 +1,35 @@
 #!/bin/bash
 
-HERE=$(dirname $(readlink -f "$0"))
-INTEGRATION_PATH=$(dirname "$HERE")
+INTEGRATION_PATH=$(git rev-parse --show-toplevel)
 export INTEGRATION_PATH
 
 DOWNLOAD_REQUIREMENTS="true"
 
-COMPOSE_CMD_BASE="docker-compose -p backend-tests \
-        -f $INTEGRATION_PATH/backend-tests/docker/docker-compose.backend-tests.yml
-        "
-COMPOSE_CMD_OPEN="docker-compose -p backend-tests \
+COMPOSE_FILES_BASE="-f $INTEGRATION_PATH/backend-tests/docker/docker-compose.backend-tests.yml"
+COMPOSE_FILES_OPEN=" \
```

Comment:
We could make `COMPOSE_FILES_OPEN` extend `COMPOSE_CMD_BASE` instead of having `backend-tests/docker/docker-compose.backend-tests.yml` in the two lists.

Reply:
The `docker-compose.backend-tests.yml` needs to be the last compose file in the list since it's an override. Do you want me to append `_BASE` to `_OPEN`?

Reply:
It was a minor suggestion. On the high level it makes sense to inherit like `BASE` > `OS` > `ENTERPRISE`. But if this complicates the script let's leave it as it (as it was).